src/main/java/software/amazon/awssdk/crt/io/ExponentialBackoffRetryOptions.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0.
+ */
+
+package software.amazon.awssdk.crt.io;
+
+import software.amazon.awssdk.crt.CrtResource;
+
+public class ExponentialBackoffRetryOptions extends CrtResource {
```

Comment:
Also, I would strongly consider not even making these a crt resource and instead create down in the s3 client C binding constructor.  These seem like settings only, I don't see why you'd ever want to share them between more than one object, and deriving from CrtResource is a serious "commitment" to ref-counting properly and the more we can avoid doing so the better off we'll be issue-wise.

Reply:
They are no longer CRT resources!